### PR TITLE
Add What-We-Use file in tooling [SATURN-1733]

### PR DESCRIPTION
**Pull request by Shakespeared**
Add master list of 'what we use' as a directory of docs and links in the tooling section. Definitely open to comments/suggestions about if this is the right format or organization, just wanted to get something in there.

### CODE REVIEW

File: tooling/What-We-Use.md
Comment (ehanna4) [inline tooling/What-We-Use.md]: The Kanban board is not currently in use so I would recommend removing 'kanban' from the description here, and maybe say like 'Sprint board and backlog' instead?

**Review by s-rubenstein — COMMENTED** [review]
I like this idea, but it kinda feels like it overlaps with our Table of Contents. Do you think this provides new information on top of that?

File: tooling/What-We-Use.md
Comment (ehanna4) [inline tooling/What-We-Use.md]: Two more additions which I'm not sure are fully team sanctioned but I have seen at least a few members use are: Insomnia and ITerm2

File: tooling/What-We-Use.md
Comment (Shakespeared) [reply tooling/What-We-Use.md]: Is Jira itself not a Kanban board? I'd always heard "Kanban" as a general term for that kind of ticketing system/display board, but maybe that's not right. I agree that having slightly more detail in the description would be good.

File: tooling/What-We-Use.md
Comment (Shakespeared) [reply tooling/What-We-Use.md]: @blrubenstein Yeah there is definitely an argument for merging this with the TOC. My thought process was that it might be nice to have it in the tooling directory for clarity, but we could also just make a 'tooling' section of the TOC, which now that I'm thinking of it, might be neater overall. 

We could also have different sections for 'team tools' and 'optional tools,' thoughts about that?

File: tooling/What-We-Use.md
Comment (ehanna4) [reply tooling/What-We-Use.md]: Kanban and Scum are two styles of Agile, and Jira supports boards for both of these. We had a Kanban board for awhile and I think it's still linked in our Jira Project which is why it might be confusing to have Kanban in the description for Jira, as we should only be using the Sprint/Scrum board at this point.

Comment (Shakespeared) [reply tooling/What-We-Use.md]: Gotcha, I didn't realize it had a more specific usage! I'll update that.

File: tooling/What-We-Use.md
Comment (Shakespeared) [reply tooling/What-We-Use.md]: Made a new commit with some updates, I moved the 'what we use' list to the readme/TOC, which I think is better, and a few other small changes. Thanks for the suggestion on that!

File: tooling/Slack-channels.md
Comment (ehanna4) [inline tooling/Slack-channels.md]: I think some sort of clarifiction about #saturn vs #saturn-team would be good here. Something like:

"#saturn-team is the private team channel for team discussion and alerting other members about absences etc. while #Saturn is a public channel where other teams can bring questions to the Saturn team."

File: tooling/Slack-channels.md
Comment (ehanna4) [inline tooling/Slack-channels.md]: also just to be nit-picky I would recommend making 'Saturn-team' and 'Saturn' lowercase as that's slack's convention and the other channels in this list are lowercased

File: tooling/Slack-channels.md
Comment (s-rubenstein) [inline tooling/Slack-channels.md]: Optional: It may be worth adding `workbench-userliaison` to this list? (I mention it in the User Liaison doc, so, probably not necessary since you should only need to watch the channel if you are in that role.)

File: tooling/Slack-channels.md
Comment (Shakespeared) [reply tooling/Slack-channels.md]: Thanks for this, that's a good call! I'll change the formatting and add some descriptions for the channels.

File: tooling/Slack-channels.md
Comment (Shakespeared) [reply tooling/Slack-channels.md]: That makes sense, I'll add it! I put in a quick description as well, but you should take a look to make sure it's accurate/complete since I'm not too familiar with the user liaison procedure.

File: tooling/Slack-channels.md
Comment (Shakespeared) [reply tooling/Slack-channels.md]: I added descriptions to the #saturn and #saturn-team channels, as well as a few others. Some of them I wasn't quite clear enough on the details to annotate (e.g. the nuances of #dsp-engineering, and #dsp-workbench, and the channel descriptions aren't very helpful). I put a TODO on it, but feel free to put suggestions here or in slack and I can drop them in if you want.

File: tooling/Slack-channels.md
Comment (ehanna4) [reply tooling/Slack-channels.md]: If you're adding `workbench-userliaison` I would recommend maybe adding like a "priveleged" or "extra" section and include that and also the on-call channels ( I will get those for you in a sec)

Comment (Shakespeared) [reply tooling/Slack-channels.md]: Maybe I should break it out into sections, like 'team/work channels' 'social channels' 'extra channels' (I'm not sure 'extra' is the best descriptor but I can't think of what else to call it).

Comment (ehanna4) [reply tooling/Slack-channels.md]: yes I like that idea !

File: tooling/Slack-channels.md
Comment (ehanna4) [inline tooling/Slack-channels.md]:
```suggestion
|\#prometheus|channel for DSP coffee talk|
```

File: tooling/Slack-channels.md
Comment (ehanna4) [inline tooling/Slack-channels.md]:
```suggestion
|\#105b| channel for building related news and updates |
```

File: tooling/Slack-channels.md
Comment (s-rubenstein) [inline tooling/Slack-channels.md]: Nit: I'd rename the file `Slack.md` Just given that it covers more than just the channels.

**Review by s-rubenstein — APPROVED** [review]
LGTM Pending one nit.

**Pull request merged**
